classes/class-p4-master-site.php

```diff
@@ -819,4 +820,24 @@ public function add_help_sidebar() {
 			$screen->set_help_sidebar( $sidebar );
 		}
 	}
+
+	/**
+	 * Filter and add class to GDPR consent checkbox label after the GDPR fields appended to comment form submit field.
+	 *
+	 * @param string $submit_field The HTML content of comment form submit field.
+	 * @param array  $args         The arguments array.
+	 *
+	 * @return string HTML content of comment form submit field.
+	 */
+	public function gdpr_cc_comment_form_add_class( $submit_field, $args ) {
+
+		$pattern[0]     = '/(for\=["|\']gdpr-comments-checkbox["|\'])/';
```

Comment:
pipe '|' is not needed in the character classes, you can remove it. It will match single or double quote without it.

Reply:
@kirdia - I amended the commit.